universitas-negeri-semarang-fakultas-matematika-dan-ilmu-pengetahuan-alam.csl

```diff
@@ -149,8 +149,8 @@
               </else>
             </choose>
           </if>
-          <else-if type="thesis">
-            <text variable="genre" suffix="."/>
+          <else-if type="thesis"><!--Note digunakan untuk mensitasi skripsi, thesis dan disertasi-->
+            <text variable="note" suffix="."/>
```

Comment:
Yeah right. Actually I am confuse. Because I don't know how to generate the genre text box in mendeley desktop for thesis type. Would you like to give me the URL of documentation to generate it?